examples/managementcli/managementcli.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/descope/go-sdk/descope"
+	"github.com/descope/go-sdk/descope/mgmt"
+	"github.com/descope/go-sdk/descope/utils"
+	"github.com/spf13/cobra"
+)
+
+// Command line flags
+
+var flags struct {
+	Identifier string
+	Email      string
+	Phone      string
+	Name       string
+	Tenants    []string
+	Domains    []string
+}
+
+// Descope SDK
+
+var descopeClient *descope.DescopeClient
+
+func prepare() (err error) {
+	if utils.GetProjectIDEnvVariable() == "" {
```

Comment:
Maybe we can ask for it.. and if provided save to env file (and put it in gitignore)

Reply:
👍